tests/integration/integration-test.py

```diff
@@ -7,6 +7,10 @@
 
 import utils
 
+err_obj = 'ErrorObject'
+park_res = 'ParkResource'
+owner_res = 'OwnerResource'
```

Comment:
If you want to define them as [constant variables](https://www.python.org/dev/peps/pep-0008/#constants), based on flake8/pep8 style guild, they must be all capital letters with underscores separating words.

> Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL.